go/vt/vtadmin/cluster/cluster.go

```diff
@@ -46,8 +46,10 @@ import (
 	"vitess.io/vitess/go/vt/vtadmin/vtadminproto"
 	"vitess.io/vitess/go/vt/vtadmin/vtctldclient"
 	"vitess.io/vitess/go/vt/vtadmin/vtsql"
+	"vitess.io/vitess/go/vt/vtctl/schematools"
 
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/proto/vtadmin"
```

Comment:
Should use vtadminpb just to be standard.

Reply:
whoops, good eye